pkg/autoscaler/sample_size.go

```diff
@@ -34,7 +34,7 @@ const (
 // if N <= 3:
 //   n = N
 // else:
-//   n = N*X / (N + X – 1), X = C^2 ­* σ^2 / MOE^2,
+//   n = N*X / (N + X – 1), X = C^2 * σ^2 / MOE^2,
```

Comment:
What actually changed here?

Reply:
There was some non printable character, which showed in `vi`